config/config.js

```diff
@@ -57,8 +50,14 @@ checkConfig('CLIENT_ID');
 checkConfig('CLIENT_SECRET');
 
 function checkConfig(setting) {
+  // If setting undefined, throw error
   if (!nconf.get(setting)) {
-    throw new Error('You must set the ' + setting + ' environment variable or' +
-      ' add it to config.json!');
+    throw new Error(`You must set the ${setting} environment variable or add it to config.json!`);
+  }
+  // If setting includes a space, emit warning
+  if (nconf.get(setting).match(/\s/)) {
+    console.warn(`The ${setting} environment variable is suspicious ("${nconf.get(setting)}")`);
```

Comment:
Possibly, although I think whitespace at the start or end is at least suspicious?